packages/sd-jwt/src/SdJwtOptions.ts

```diff
@@ -0,0 +1,28 @@
+import type { HashName, Key } from '@aries-framework/core'
+import type { DisclosureFrame } from 'jwt-sd'
+
+export type SdJwtCreateOptions<Payload extends Record<string, unknown> = Record<string, unknown>> = {
+  issuerDid: string
+  issuerKey: Key
+  disclosureFrame?: DisclosureFrame<Payload>
+  holderBinding: string | Key
+  hashingAlgorithm?: HashName
+}
+
+export type SdJwtReceiveOptions = {
+  issuerDid: Key
+  holderKey?: Key
```

Comment:
These will both already be in the JWT right?

Reply:
I changed it to `issuerKey` and `holderkey` which are used to confirm the signature and check whether the `cnf` claim is correct. If this info is invalid the sd-jwt cannot be used.

Reply:
now using `issuerDidUrl` and `holderDidUrl`.